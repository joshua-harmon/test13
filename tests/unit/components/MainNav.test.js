import { mount } from "@vue/test-utils";
import MainNav from "@/components/MainNav.vue";

describe("MainNav", () => {
	it("displays company name", async () => {
		const wrapper = mount(MainNav);
		await wrapper.setData({ company: "Booboo Careers" });
		expect(wrapper.text()).toMatch("Booboo Careers");
	});
});
