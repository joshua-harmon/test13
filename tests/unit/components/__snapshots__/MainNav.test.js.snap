// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainNav displays company name: Booboo Careers 1`] = `"Booboo Careers"`;

exports[`MainNav displays company name: Boogle Careers 1`] = `"Boogle Careers"`;
